exp-player/addon/components/exp-player.js

```diff
@@ -142,6 +143,7 @@ able to continue the study.
         next() {
             console.log('next');
             var frameIndex = this.get('frameIndex');
+            this.sendAction('updateFrameIndex', frameIndex);
```

Comment:
That will break other usages of `exp-player`.

In some ways `exp-player` deviates from proper data-down-actions-up paradigms since it mutates `frameIndex` internally rather than calling an action on the parent controller/component. Since `frameIndex` is a bound property, I'd expect in your case that the controller's value would update too (without needing to send the `updateFrameIndex` action). But I'm guessing you tried this and found that wasn't the case?

On approach would be to generalize this behavior into an `exp-player-controller` mixin like the [`exp-route-mixin`](https://github.com/CenterForOpenScience/exp-addons/blob/develop/exp-player/addon/mixins/exp-player-route.js) that exposes actions for changing the `frameIndex` outside of exp-player. Might be worth looking into, but first can you confirm that `frameIndex` isn't already getting changed on the controller (without the new action)?